keyboards/40percentclub/4pack/readme.md

```diff
@@ -0,0 +1,17 @@
+# 4 Pack
+
+![4pack](https://i.imgur.com/rioXXq5l.jpg)
+
+A 4-key macropad PCB with its switch and LED pins wired directly to microcontroller IO pins.
+
+You can find the main blog post about this macropad [here](http://www.40percent.club/2017/07/4-pack.html), also the Gerber files and other documentation is located [here](https://git.40percent.club/di0ib/Misc/src/branch/master/4pack).
+
+Keyboard Maintainer: [Arda Kilicdagi](http://github.com/ardakilic), [The QMK Community](https://github.com/qmk)  
+Hardware Supported: 4 Pack PCB, Pro Micro  
+Hardware Availability: [40percent.club](https://git.40percent.club/di0ib/Misc/src/branch/master/4pack)
+
+Make example for this macropad (after setting up your build environment):
+
+    make 40percentclub/4pack:default:avrdude
```

Comment:
We have a universal flashing mechanic now, that reads the config and will auto-detect which bootloader it's using.  

```suggestion
    make 40percentclub/4pack:default:flash
```

Reply:
Updated in my last commit, thanks for the heads-up.